CHANGELOG.rst

```diff
@@ -18,6 +18,18 @@ Changelog
 * Removed the deprecated ``CAST5``, ``SEED``, ``IDEA``, and ``Blowfish``
   classes from the cipher module. These are still available in
   :doc:`/hazmat/decrepit/index`.
+* Make instances of
+  :class:`~cryptography.hazmat.primitives.hashes.HashAlgorithm` as well as
+  instances of classes in
+  :mod:`~cryptography.hazmat.primitives.asymmetric.padding`
+  comparable.
+* Added `salt_length` property to
```

Comment:
Let's put these in a separate PR. Whether or not we want to do `__eq__` is possibly debateable, but if you want these properties to be public that's a separate discussion.

Reply:
Hi @reaperhulk 

Thanks for the feedback - I've updated this PR to only include the __eq__ instances. I'll make a separate PR for the extra properties.

kr, Mat